lib/puppet/provider/pulp_rpmbind/consumer.rb

```diff
@@ -0,0 +1,41 @@
+Puppet::Type.type(:pulp_rpmbind).provide(:consumer) do
+  desc 'Bind/unbind to an RPM repo'
+
+  confine osfamily: :redhat
+  commands consumer: '/bin/pulp-consumer'
+  commands grep:     '/bin/grep'
+
+  def self.instances
+    begin
+      binds = grep('-oP', '^\[\K.*(?=\]$)', '/etc/yum.repos.d/pulp.repo')
```

Comment:
So `/etc/yum.repos.d/pulp.repo` is the [default](https://github.com/pulp/pulp/blob/764fe65af86c72996a109a4769e7644b274b29d8/client_consumer/pulp/client/consumer/config.py#L26) in pulp , but it *is* possible to override it in `/etc/pulp/consumer/consumer.conf` and this module does even expose that [option](https://github.com/Katello/puppet-pulp/blob/a48570999513e3f2cae5a4e84a373e836e048850/manifests/consumer/params.pp#L21).

I'm not sure how many people actually do this, (maybe none), but I suppose I should code for it.